### PR TITLE
update faraday requirement to be larger or equal to 1.1

### PR DESCRIPTION
**Pull request by autonomous**
prevent metainspector from locking the version of faraday to 1.1.x

### CODE REVIEW

File: meta_inspector.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.version       = MetaInspector::VERSION
 
   gem.add_dependency 'nokogiri', '~> 1.11.0'
-  gem.add_dependency 'faraday', '~> 1.1.0'
+  gem.add_dependency 'faraday', '>= 1.1.0'
```
Comment (jaimeiniesta) [inline meta_inspector.gemspec:18]: How about this?

```
gem.add_dependency 'faraday', '~> 1.1'
```

This way we can allow minor releases like 1.4, but avoid major releases like 2.0 that may have incompatible API changes.

**Review by jaimeiniesta — CHANGES_REQUESTED** [review]
Thanks! Let's relax this dependency but still follow [semantic versioning](https://semver.org/) instead, what do you think?

**Comment by jaimeiniesta** [comment]
Closed in favor of https://github.com/metainspector/metainspector/pull/280

**Comment by jaimeiniesta** [comment]
Release 5.11.1 relaxes the dependencies.

Thanks!

File: meta_inspector.gemspec (hunk 1 above)
Comment (autonomous) [reply meta_inspector.gemspec:18]: Great suggestion @jaimeiniesta - also, thanks for #280 🎉

**Pull request closed without merging**